editor/editor_inspector.h

```diff
@@ -35,6 +35,7 @@
 #include "scene/gui/button.h"
 #include "scene/gui/dialogs.h"
 #include "scene/gui/line_edit.h"
+#include "scene/gui/option_button.h"
```

Comment:
Lots of useless includes here that could be replaced with forward-declarations :/

Reply:
This should be cleaned up all over the editor at some point.